docs(patternfly-accessibility): Updates content as part of content audit. (#3772)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/developer-resources/patternfly-accessibility.md

section: accessibility
---

> # PatternFly components accessibility 

Reviewer: This seems like it'd be better as a level 2 heading. Also, could we omit the "PatternFly" from these two headings? Since we already have the "PatternFly's accessibility" heading, shortening these to "Component accessibility" and "Accessibility standards"

> # PatternFly components accessibility 

Author: ```suggestion
## Component accessibility 
```

We do our best to build accessibility into PatternFly's components to take some of the accessibility work off your hands.

As PatternFly is updated and enhanced, we maintain  accessibility through a combination of automated and manual testing. We use [aXe: The Accessibility Engine](https://www.deque.com/axe/) to ensure that all components pass an accessibility audit before they’re added to PatternFly. 

> We also regularly audit keyboard accessibility with both manual testing and integration tests. We aim to provide full support for Voice Over as our main screen reader, but we still test our components through NVDA and JAWS. As part of our manual audit, every component is run through Voice Over to make sure they’ll be as accessible as possible in your products.

Reviewer: Small nit (removing the space between Voice and Over):

```suggestion
We also regularly audit keyboard accessibility with both manual testing and integration tests. We aim to provide full support for VoiceOver as our main screen reader, but we still test our components through NVDA and JAWS. As part of our manual audit, every component is run through VoiceOver to make sure they’ll be as accessible as possible in your products.
```

> ## PatternFly accessibility standards

Author: ```suggestion
## Accessibility standards
```

> Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly:

Reviewer 2: Super small observation/potential typo, I think that this last sentence is missing a word, potentially 'in'? I'm thinking it might be, "If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect in PatternFly:"

Author: Ah yes, ty for catching! I pushed a change - went with "expect PatternFly to adhere to" if that sounds good?

Reviewer 2: Sounds good to me! 🙂 👍

> Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly:

Author: ```suggestion
Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly to adhere to:
```

> Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, these are the accessibility guidelines you can expect PatternFly to meet: 

Author: ```suggestion
Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, these are the accessibility guidelines you can expect PatternFly to meet the following: 
```

> Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, these are the accessibility guidelines you can expect PatternFly to meet the following: 

Author: ```suggestion
Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly:
```

> Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, these are the accessibility guidelines you can expect PatternFly to meet the following: 

Reviewer: The second sentence sounds a bit off? wdyt of something like, "If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly to meet:"
